prediction_market_agent/agents/microchain_agent/deploy.py

```diff
@@ -163,6 +163,8 @@ class DeployableMicrochainModifiableSystemPromptAgent3(
 ):
     task_description = AgentIdentifier.MICROCHAIN_AGENT_OMEN_LEARNING_3
     model = SupportedModel.llama_31_instruct
+    # Force less iterations, because Replicate's API allows at max 4096 input tokens.
```

Comment:
This is unfortunate, as officially Llama 3.1 405b should support a much bigger context window.

![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/e83d025a-6bbe-4dee-8e02-ea792c1c087a)

(from Replicate dashboard)